GrandCentralBoard/Widgets/Bonus/Scene/BubbleScalingController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  BubbleScalingController.swift
+//  GrandCentralBoard
+//
+//  Created by Bartłomiej Chlebek on 07/04/16.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import SpriteKit
+
+protocol BubbleScalingControllerDelegate: class {
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeUp spriteNode: SKSpriteNode)
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeDown spriteNode: SKSpriteNode)
+}
+
+extension BubbleScalingControllerDelegate {
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeUp spriteNode: SKSpriteNode) { }
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeDown spriteNode: SKSpriteNode) { }
```

Comment:
Do we really need that? If these method are optional (and because of that they aren't used in the current implementation) just remove them from the delegate's protocol.